### ISSUE TITLE
some suggestions based on my own experience

### ISSUE THREAD

**Issue by crazyhottommy** [JasonJWilliamsNY/abstracts#1]
Hi Jason,

Looking forward to see your presentation. I have several suggestions:

1. Naming the files.
This is the number 1 lesson everyone should learn.
see a [PPT](https://rawgit.com/Reproducible-Science-Curriculum/rr-organization1/master/organization-01-slides.html) by Jenny Bryan.

2. beware that excel can make mistakes: https://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-1044-7

and it is hard to parse if the entries are not entered properly and those (e.g.highlighting) formatted things can be a barrier to read into R/python.

3. data cleaning skills
data janitor work costs most time in a data project.
https://www.nytimes.com/2014/08/18/technology/for-big-data-scientists-hurdle-to-insights-is-janitor-work.html

4. Project organization/automation 
organize your project to make your work more reproducible. use R project if you use Rstudio.
and have a folder structure.

some refs:
http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1000424
https://www.biorxiv.org/content/early/2017/07/24/167619
https://arxiv.org/abs/1609.00037

A really nice one on sequencing data project https://www.biorxiv.org/content/early/2017/08/29/136358

I have some other notes https://github.com/crazyhottommy/getting-started-with-genomics-tools-and-resources#automate-your-workflow-open-science-and-reproducible-research

Tommy

**Comment by JasonJWilliamsNY** [comment]
Thanks will definitely look to comment om these kinds of issues!
